core/domain/feedback_jobs_continuous_test.py

```diff
@@ -147,16 +263,17 @@ def test_multiple_threads_multiple_exp(self):
             ModifiedFeedbackAnalyticsAggregator.start_computation()
             self.assertEqual(self.count_jobs_in_taskqueue(), 1)
             self.process_and_flush_pending_tasks()
+            feedback_analytics = (
+                ModifiedFeedbackAnalyticsAggregator.get_thread_analytics_multi(
+                    [exp_id_1, exp_id_2]))
```

Comment:
Actually if both this and the previous test are testing the multi() version of the function, then I don't really see what the difference is.

Maybe revert this to what it was before and rename the tests to test_get_multi_for_multiple_threads_and_exps() and test_get_for_multiple_threads_and_exps()?

In fact, you can alternatively put them together. Have a common setup, and then do the calls to get_thread_analytics() and check the results, then do calls to get_thread_analytics_multi() and check the results. That way, you check both methods whilst also reducing duplication. What I was objecting to before was that you were removing the test for the non-multi case.